.github/workflows/entropy-check.yml

```diff
@@ -22,24 +22,24 @@ jobs:
       - name: Install python env
         uses: ./.github/actions/install-python-env
         with:
-          fetch-depth: 0 # Ensure the action respects the desired fetch depth
+          fetch-depth: 0
 
         # Determine the PR branches and set environment variables
-      - name: Set PR Branch Environment Variables
-        env:
-          PR_BRANCH: ${{ github.head_ref }}
-          MAIN_BRANCH: ${{ github.base_ref }}
+      - name: Set and Display PR Branch Environment Variables
         run: |
-          "echo 'PR_BRANCH=${PR_BRANCH}' >> $GITHUB_ENV"
-          "echo 'MAIN_BRANCH=${MAIN_BRANCH}' >> $GITHUB_ENV"
+          export PR_BRANCH=${{ github.head_ref }}
```

Comment:
Since these are github values you're referencing, would it be possible for you to reference them directly instead of re-setting them as environment values?